pyvista/core/grid.py

```diff
@@ -254,8 +254,9 @@ def _from_arrays(
         # Ensure dimensions are properly set
         self._update_dimensions()
 
+    # typing depends on numpy version
```

Comment:
This method is declared as a simple wrapper for `np.meshgrid`, if we do this, it is no longer technically true.  It is a wrapper that enforces a return type. I think the practical importance is pretty low in either case.  I've heard two votes for converting to tuple always, I'll wait for more feedback before implementing it though.

Reply:
To me the wrapper reference is just about semantics rather than strict API adherence. I'd expect any user actually affected by this to benefit from removing this moving target. We could rephrase the docs as "implemented as a thin wrapper for meshgrid".

Reply:
I think the practical importance of the change is not important.  I took the advice to be opinionated and return `tuple` type.